assets/scripts/main.js

```diff
@@ -46,11 +46,13 @@ class App {
   onHashHandle() {
     const {hash} = location;
     let el = document.createElement('link');
-    el.rel = 'stylesheet';
-    el.href = '/static/print.css';
-    document.head.appendChild(el);
-    document.body.classList.toggle('print', hash === '#print');
-    let printView = new Print('print-view');
+    if (hash === '#print') {
+      el.rel = 'stylesheet';
+      el.href = '/static/print.css';
+      document.head.appendChild(el);
+      document.body.classList.toggle('print', hash === '#print');
```

Comment:
This can be an `add`, as the `if` above already makes sure that it is `hash === #print`.